kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -321,21 +352,26 @@ class NotificationManager internal constructor(
                     is NotificationButton.PLAY_PAUSE -> {
                         PlaybackStateCompat.ACTION_PLAY or PlaybackStateCompat.ACTION_PAUSE
                     }
+
                     is NotificationButton.BACKWARD -> {
                         rewindIcon = button.icon ?: rewindIcon
                         PlaybackStateCompat.ACTION_REWIND
                     }
+
                     is NotificationButton.FORWARD -> {
                         forwardIcon = button.icon ?: forwardIcon
                         PlaybackStateCompat.ACTION_FAST_FORWARD
                     }
+
                     is NotificationButton.SEEK_TO -> {
                         PlaybackStateCompat.ACTION_SEEK_TO
                     }
+
```

Comment:
Please don't intermingle stylistic changes with bug fixes. Can you toll all these style changes back?